### PR TITLE
feat: add accommodation availability column

### PR DESCRIPTION
**Pull request by mirsadikov**
added accommodation availability column

### CODE REVIEW

File: src/accommodation/dto/create-accommodation.dto.ts [hunk 1]
```diff
@@ -50,6 +51,10 @@ export default class CreateAccommodationDto {
   @Max(ACCOMMODATION_MAX_PRICE ? +ACCOMMODATION_MAX_PRICE : 2147483647)
```
Comment (gitkoshelew) [inline src/accommodation/dto/create-accommodation.dto.ts]: create constant instead for hardcoded value

File: src/i18n/ru/errors.json [hunk 2]
```diff
@@ -79,5 +79,6 @@
   "PAYMNET_FAILED_WHILE_CONFIRM": "Ошибка подтверждения платежа",
   "NOT_FOUND_HOST_PROFILE": "Профиль хоста не найден",
   "HOST_PROFILE_FAILED_TO_GET": "Не удалось получить профиль хоста",
-  "REVIEWS_FAILED_TO_GET_LIST": "Не удалось получить список отзывов."
+  "REVIEWS_FAILED_TO_GET_LIST": "Не удалось получить список отзывов.",
+  "BAD_REQUEST_BOOKING_NOT_AVAILABLE": "Жилье недоступно для бронирования"
```
Comment (gitkoshelew) [inline src/i18n/ru/errors.json]: Change maybe this translation like 'Аппартаменты времменно недоступны для бронирования"  to explain that it's temporary.

Comment (gitkoshelew) [inline src/i18n/ru/errors.json]: Change maybe this translation like 'Аппартаменты времменно недоступны для бронирования"  to explain that it's temporary.

**Review by gitkoshelew — APPROVED** [review]

**Review by safyurtago — APPROVED** [review]
approved

**Review by Zach13131 — APPROVED** [review]

**Pull request merged**